packages/url_launcher/url_launcher/android/src/main/java/io/flutter/plugins/urllauncher/WebViewActivity.java

```diff
@@ -38,6 +39,11 @@ public void onReceive(Context context, Intent intent) {
   private final WebViewClient webViewClient =
       new WebViewClient() {
 
+        /*
+         * This method is deprecated in API 24. Still overridden to support
+         * earlier Android versions.
+         */
+        @SuppressWarnings("deprecation")
```

Comment:
TBH looking at this snippet now I'm confused at all as to why we override this method in order to directly call loadUrl on older Android versions?

I left a question on the PR that added this though it was a year ago so not sure if the author will still be able to provide info: https://github.com/flutter/plugins/pull/1848/files#r485141991